flink-autoscaler/src/main/java/org/apache/flink/autoscaler/event/AutoScalerEventHandler.java

```diff
@@ -32,32 +35,45 @@
 import static org.apache.flink.autoscaler.metrics.ScalingMetric.TRUE_PROCESSING_RATE;
 
 /**
- * Handler for autoscaler events.
+ * Handler for autoscaler events, it logs events by default.
  *
  * @param <KEY> The job key.
  * @param <Context> Instance of JobAutoScalerContext.
  */
 @Experimental
-public interface AutoScalerEventHandler<KEY, Context extends JobAutoScalerContext<KEY>> {
-    String SCALING_SUMMARY_ENTRY =
+public class AutoScalerEventHandler<KEY, Context extends JobAutoScalerContext<KEY>> {
```

Comment:
I think this should remain an interface because this is meant to be implemented by other backends. We already have two implementation of this: Kubernetes, Standalone, and Logging only.

Reply:
Both are fine to me, updated the `AutoScalerEventHandler` as an interface and added a `LoggableEventHandler` as the logging  implementation.